guide_sidebar.py

```diff
@@ -0,0 +1,399 @@
+            <text style="margin: 5px; font-size: 20px;">Obama</text>
+            <div style="background-color: #E4AA7A; width: 20px; height: 20px; margin: 0;"></div>
+            <text style="margin: 5px; font-size: 20px;">Trump</text>
+        </div>
+    </div>
+    """,
+        unsafe_allow_html=True
+    )
+
+    st.markdown("<h2>Facility Inspections - CAA, CWA, RCRA</h2> <subtitle>Mostly complete data<subtitle>",
+                unsafe_allow_html=True)
+    # this is a filler graph
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+    st.markdown("<h2>Facility Inspections - CAA, CWA, RCRA</h2> <subtitle>Potentially incomplete data<subtitle>",
+                unsafe_allow_html=True)
+    # this is a filler graph
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+    st.markdown("<h2>Facility Inspections - CAA, CWA, RCRA</h2> <subtitle>Mostly incomplete data<subtitle>",
+                unsafe_allow_html=True)
+    # this is a filler graph
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+
+
+def comparisonTabInfo():
+    # Define the CSS style for the gray rectangle
+    gray_rectangle_style = """
+        background-color: #e6e6e6;
+        padding: 10px;
+        border-radius: 5px;
+        color: black;
+    """
+
+    # Define the HTML content as a string
+    html_content = f"""
+    <div style="{gray_rectangle_style}">
+        <ul>
+            <li>These two charts show how inspections and violations in this state 
+            compare to the national average per 1000 facilities in 2022</li>
+            <li>We use data from 2022 as it was the most recent full year and 
+            the ECHO database only reports currently active facilities</li>
+            <li>To enable comparison across locations with a differing number 
+            of active facilities, we standardize the comparison to a value per 
+            1000 facilities, proportionally adjusting the data if there are more 
+            or less than 1000 facilities in a district or state</li>
+        </ul>
+    </div>
+    """
+
+    # Use st.markdown to render the HTML content
+    st.markdown(html_content, unsafe_allow_html=True)
+
+
+def comparisonTabData():
+    st.markdown("<h2 style='text-align: center'>Inspections per 1000 Facilities (2022)</h2>",
+                unsafe_allow_html=True)
+    # this is where graph for CAA Violators
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+    st.markdown("<h2 style='text-align: center'>Violations per 1000 Facilities in 2022</h2>",
+                unsafe_allow_html=True)
+    # this is where graph for CWA Violators
+    chart_data = pd.DataFrame(np.random.randn(20, 3), columns=["a", "b", "c"])
+    st.bar_chart(chart_data)
+
```

Comment:
This is just a small suggestion. I liked how you guys have the functions for each tab placed right on top of each other. It would be even nicer to also have comments in between each to specify what the following couple of functions are for. For instance, you can add "#the following 2 functions, non_complianceTabInfo() and non_complianceTabData() are the content for the noncompliance tab"